es/criteria/document-maturity.md

```diff
@@ -22,20 +22,20 @@ Señalar claramente la madurez de una codebase ayuda a otros a decidir si reutil
 
 * Garantizar que otros usarán el código.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"